hazelcast/src/jepsen/hazelcast.clj

```diff
@@ -386,15 +388,24 @@
                            :workload checker})
             :model      model})))
 
+(defn workload-names
+  "Comma separated list of available test"
+  []
+  (str/join ", " (map name (keys (workloads nil)))))
+
+
 (def opt-spec
   "Additional command line options"
-  [[nil "--workload WORKLOAD" "Test workload to run, e.g. atomic-long-ids."
-    :parse-fn keyword]])
+  [[nil "--workload WORKLOAD" (str "Test workload to run, possible options are " (workload-names) "Required.")
+    :parse-fn keyword
```

Comment:
Like I mentioned in the previous review, you want to use `(comp (workloads) keyword)` as the parse fn. That way the option parser will look up the workload for you.

Reply:
Oh, wait, I think I'm looking at the wrong version of the code here. Ummmmm... hmm. This might be because (workloads) is a function of test, and we're passing in nil. I'll go poke at the code a bit.